README.md

```diff
@@ -80,14 +80,17 @@ vim.o.completeopt = "menu,menuone,noselect"
 - `compe.incomplete_delay (number)`: Delay for LSP's `isIncomplete`. default: `400`
 - `compe.allow_prefix_unmatch`: TODO???
 
-- `compe.source.path (bool)`: Path completion. default: `false`
-- `compe.source.buffer (bool)`: Buffer completion. default: `false`
-- `compe.source.vsnip (bool)`: Vsnip completion, make sure you have `vim-vsnip` installed. default: `false`
-- `compe.source.nvim_lsp (bool)`: Nvim's builtin LSP completion. default: `false`
-- `compe.source.nvim_lua (bool)`: Nvim's Lua "stdlib" completion. default: `false`
-- `compe.source.spell (bool)`: Dictionary completion if you set `spell`. default: `false`
-- `compe.source.snippets_nvim (bool)`: [snippets.nvim](https://github.com/norcalli/snippets.nvim) completion. default: `false`
-- `compe.source.your_awesome_source (table | dict)`: Override source configuration using a custom `table`(lua) or `dictionary`(vimscript).
+- `compe.source.path (bool | table | dict)`: Path completion. default: `false`
+- `compe.source.buffer (bool | table | dict)`: Buffer completion. default: `false`
+- `compe.source.vsnip (bool | table | dict)`: [vim-vsnip](https://github.com/hrsh7th/vim-vsnip) completion, make sure you have it installed. default: `false`
+- `compe.source.lamp (bool | table | dict)`: [vim-lamp](https://github.com/hrsh7th/vim-lamp) completion, make sure you have it installed. default: `false`
```

Comment:
Thank you!
But Currently, I don't advertise vim-lamp 😅
Could you remove it?

I will remove lamp source from nvim-compe builtin.

Reply:
sure :)